runway/commands/modules_command.py

```diff
@@ -51,34 +51,37 @@ def assume_role(role_arn, session_name=None, duration_seconds=None,
             'AWS_SESSION_TOKEN': response['Credentials']['SessionToken']}
 
 
-def determine_module_class(path, module_options):
+def determine_module_class(path, class_path):
     """Determine type of module and return deployment module class."""
-    if not module_options.get('class_path'):
```

Comment:
we don't need the value outside of this function, so we don't need to store it in the dict